src/Application/Actions/Donations/Confirm.php

```diff
@@ -92,9 +91,28 @@ protected function action(Request $request, Response $response, array $args): Re
             // Note that `on_behalf_of` is set up on create and is *not allowed* on update.
         ]);
 
-        $this->stripeClient->paymentIntents->confirm($paymentIntentId, [
-            'payment_method' => $pamentMethodId,
-        ]);
+        try {
+            $this->stripeClient->paymentIntents->confirm($paymentIntentId, [
+                'payment_method' => $pamentMethodId,
+            ]);
```

Comment:
I should have really thought that confirm might thrown and put in a try catch myself when I added this function call.